src/components/layout/Menu/Menu.js

```diff
@@ -1,22 +1,46 @@
-import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
-import ChevronRightIcon from "@mui/icons-material/ChevronRight";
-import { useTheme } from "@mui/material";
-import IconButton from "@mui/material/IconButton";
-import * as React from "react";
-import MenuItem from "./MenuItem/MenuItem";
-import { DrawerHeaderStyled, DrawerStyled } from "./MenuStyled";
-import TopMenu from "./TopMenu/TopMenu";
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import MenuIcon from '@mui/icons-material/Menu';
+import { useTheme } from '@mui/material';
+import IconButton from '@mui/material/IconButton';
+import Toolbar from '@mui/material/Toolbar';
+import * as React from 'react';
+import IconLogo from '../../UI/IconLogo';
+import MenuItem from './MenuItem/MenuItem';
+import {
+  AppBarStyled,
+  DrawerHeaderStyled,
+  DrawerStyled,
+  LogoStyled,
+} from './MenuStyled';
 
 const Menu = ({ open, handleDrawerClose, handleDrawerOpen }) => {
   const theme = useTheme();
 
   return (
     <>
-      <TopMenu handleDrawerOpen={handleDrawerOpen} open={open} />
```

Comment:
Why did you move the TopMenu code back to Menu.js? Is it a merge issue or you had some idea about it? I think it is good to have a separate component to keep them as thin and clean as possible. Also, the top menu contains the logout icon which is missing.